src/repositories/projectRepository.ts

```diff
@@ -437,7 +437,10 @@ export const totalProjectsPerDate = async (
   }
 
   if (onlyListed) {
-    query.andWhere(`project."reviewStatus" = 'Listed'`);
+    query.andWhere(
+      `project.statusId = ${ProjStatus.active} AND project.reviewStatus = :reviewStatus`,
```

Comment:
I think it's not a good idea to add `statusId` condition inside `onlyListed` condition, because those are separate conditions

Reply:
@RamRamez he can move it outside, but what if this _statusId_ is only related to listed projects? I just asking, want to know :)

Reply:
@kkatusic I checked and we have some listed projects with status other than `active`, so we should check them separately. I asked Mitch in the issue for clarification.

Reply:
thx

Reply:
@Reshzera I just fixed the error could you please take a look again